en/csql.rst

```diff
@@ -403,7 +403,7 @@ The **;SHELL_Cmd** command registers a shell command to execute with the **SHELL
     drwxr-xr-x  4 DBA cubrid   4096 Jul 29 16:14 cubridmanager
     csql>
 
-The OS environment variable(CUBRID_CSQL_SHELL, SHELL) may be registered and used in advance. ::
+The OS environment variable(CUBRID_CSQL_SHELL or SHELL, CUBRID_CSQL_SHELL is used if both environment variables are set) may be registered and used in advance. ::
```

Comment:
```suggestion
The OS environment variable(CUBRID_CSQL_SHELL or SHELL; CUBRID_CSQL_SHELL is used if both environment variables are set) may be registered and used in advance. ::
```

Reply:
`..._CUBRID_CSQL_SHELL or SHELL, CUBRID_CSQL_SHELL_...`
Using a comma instead of a semi colon here initially suggests that there are three options instead of two, which requires one to reread the sentence to properly understand it.

Reply:
thank you, i will apply your opinion.